.github/workflows/go.yml

```diff
@@ -24,5 +24,15 @@ jobs:
     - name: Build
       run: make build
 
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.22'
```

Comment:
Hey @elmiko , is it synced with project version? I am checking the go.mod and looks like it is in 1.23.